src/shared/components/query/QueryStore.ts

```diff
@@ -1885,6 +1895,11 @@ export class QueryStore {
                 !this.defaultMutationProfile
             )
                 return 'Mutation data query specified in OQL, but no mutation profile is available for the selected study.';
+            if (
+                this.alterationTypesInOQL.haveStructuralVariantInQuery &&
+                !this.defaultStructuralVariantProfile
+            )
+                return 'Structural Variant data query specified in OQL, but no structural variant profile is available for the selected study.';
```

Comment:
second word (variant) probably should be lower case.  Also, could it be more than one study?

Reply:
it could be more than one study

Reply:
these messages are shown for single study